src/core/Akka.Persistence.TestKit/JournalInterceptors.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace Akka.Persistence.TestKit
+{
+    using System;
+    using System.Threading.Tasks;
+
+    internal static class JournalInterceptors
+    {
+        internal class Noop : IJournalInterceptor
```

Comment:
Would you mind adding some XML-DOC comments to explain what each of these do just to help other OSS users know in the future?

Reply:
Definitely. I want to finish first with snapshots
 Because most probably I will refactor something so that it all looks as complete picture.